src/game.cpp

```diff
@@ -7754,6 +7755,7 @@ bool pet_menu(monster *z)
     if( z->type->in_species( ZOMBIE ) ) {
         pet_name = _("zombie slave");
     }
+    pet_name = z->name();
```

Comment:
Old value of `pet_name` isn't used anywhere. The code above can be removed and changed to just
```
std::string pet_name = z->name();
```